ballerina/tests/tests.bal

```diff
@@ -23,20 +23,18 @@ configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
 configurable string apiKey = isLiveServer ? os:getEnv("OPENAI_API_KEY") : "test";
 configurable string serviceUrl = isLiveServer ? "https://api.openai.com/v1" : "http://localhost:9090";
 
-
-final ConnectionConfig config = {
-    auth: {
-        token:apiKey
-    }
-};
-final Client openAIAudio = check new(config,serviceUrl);
-
 const AUDIO_FILE_PATH1 = "tests/resources/audioClip.mp3";
 
 @test:Config {
     groups: ["live_tests", "mock_tests"]
 }
 isolated function  testCreateTranslation() returns error? {
+    final ConnectionConfig config = {
+        auth: {
+            token: apiKey
+        }
+    };
```

Comment:
No need to update the test cases with function level client initializations. We use that only as a convention in our guides and documentation.